README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
+Unsere App ist sehr datenhungrig. Um das Spiel zu testen, ohne die App zu installieren, haben wir ein Postman package erstellt, welches die App immitiert mit dem sich das Spiel durchspielen lässt (die Spielerfahrung leidet darunter jedoch deutlich). Die Nutzung ist wie folgt:
+1. `detective-game-no-app-walkthrough.json` mit Postman importieren
```

Comment:
Request for `make_analayst_appointment` broken for me as `body` is sent as `raw:text`. Makes server answer `please provide json data`.

Should be fixed by changing `raw:text` to `raw:json`

Reply:
weird, it's correct in the postman package...

Reply:
If I import it in Postman it's set to text.
Does it work on your machine?
I doubt this is a problem with my Postman version

Reply:
I just tried deleting my package and reimporting it, works like a charm - so I guess it must be a problem with your Postman version... though I agree this is weird